src/Profile/Shopware/Converter/CustomerWishlistConverter.php

```diff
@@ -63,6 +63,7 @@ public function convert(array $data, Context $context, MigrationContextInterface
         $converted['customerId'] = $customerMapping['entityUuid'];
         $converted['salesChannelId'] = $shopMapping['entityUuid'];
         $converted['products'][] = [
+            'id' => $this->mappingService->getOrCreateMapping($this->connectionId, DefaultEntities::CUSTOMER_WISHLIST_PRODUCT, $data['userID'] . '_' . $data['ordernumber'], $context)['entityUuid'],
```

Comment:
Did you check if other `Wishlist` converters e.g. SW6 / Magento have the same issue in their implementation?

Reply:
no, i will check that

Reply:
SW6 -> SW6 Works fine

Reply:
Magento has no wishlists !?